write_spectra/write_spectra.py

```diff
@@ -0,0 +1,129 @@
+from astropy.table import Table
+from specutils import Spectrum1D
+from header_function import *
+import astropy.units as u
+import matplotlib.pyplot as plt
+import numpy as np
+
+fits_data_dir = '/Users/jolie/PycharmProjects/JWST_VHS1256b_Reduction/write_spectra/'  #Where you want to write fits files to
+
+#Header information about both NIRSpec and MIRI Spectra
+JWST_header_dict = {
+    'RA': '194.007637',
+    'Dec': '-12.957692',
```

Comment:
I downloaded and checked out the FITS header and dec was missing. Is it possible the dictionary keys are case sensitive?